filters/general/filter_34_HageziPersonal/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "filterId": "hagezi_personal",
+  "id": 34,
+  "name": "HaGeZi Personal Black & White",
+  "description": "An extension for existing blocklists, my personal black and whitelist. Unblocks false positives and referral domains.",
+  "timeAdded": 1667987017479,
+  "homepage": "https://github.com/hagezi/dns-blocklists",
+  "expires": "4 days",
+  "displayNumber": 4,
+  "environment": "prod",
```

Comment:
Could you please set it to `dev` for the time being? We'll update it to prod when it's tested privately

Reply:
Updated